frontend/src/modules/onboard/components/onboard-sync-data-step.vue

```diff
@@ -70,7 +70,7 @@ const store = useStore();
 const loading = computed(
   () => store.getters['integration/loadingFetch'],
 );
-const integrationsArray = computed(() => CrowdIntegrations.mappedConfigs(store)
+const integrationsArray = computed(() => CrowdIntegrations.mappedEnabledConfigs(store)
   .filter((i) => !i.onboard?.highlight && !!i.onboard));
 const highlightedIntegrationsArray = computed(() => CrowdIntegrations.mappedConfigs(store)
```

Comment:
Can you also update here to `mappedEnabledConfigs`?

Reply:
Yeah